### DISCUSSION TITLE
How do I use Farcaster from the web or with an open-source app?

### DISCUSSION THREAD

**Discussion by alexgleason** [farcasterxyz/protocol#153] [Q&A] [answered]
It seems like Farcaster is only available as a proprietary mobile app. Is there a more open way to access it?

I'm a developer of Nostr and ActivityPub. I'm interested in checking it out.

**Comment by alexgleason** [comment]
Okay, I found this: https://github.com/a16z/awesome-farcaster But are ANY of these apps open source?

**Reply by AXDevTimothy** [reply]
I am interested in this as well.  I have a degoogled graphene OS phone and I cannot use warpcast without a google or apple account it seems.

**Comment by horsefacts** [comment, marked as answer]
I recommend [Herocast](https://app.herocast.xyz/) ([Github](https://github.com/hellno/herocast)) if you're looking for an open-source web client.
